cfgov/v1/jinja2tags/__init__.py

```diff
@@ -68,7 +69,8 @@ def is_filter_selected(context, fieldname, value):
 
     # Dirty hack to check the default option for the `archived` filter
     if fieldname == 'archived' and value == 'include':
-        return True
+        if flag_enabled('ARCHIVE_FILTER_OPTIONS'):
+            return True
```

Comment:
I don't think this is necessary. It's sufficient to ensure that the field doesn't appear.

Reply:
Good catch. This is not needed.